docs/run/sessions.mdx

```diff
@@ -132,17 +135,52 @@ words, a deactivated session does not block the creation of other
 sessions. Jobs from this new session would then take priority until it
 is deactivated or closed, at which point normal job selection resumes.
 
+To find the interactive timeout value for a session, follow the instructions in [Determine session details](#session-details).
+
+<span id="session-details"></span>
+## Determine session details
+
+You can find details about a session by using the `session.details()` method, from the [Quantum Platform Jobs page](https://quantum-computing.ibm.com/jobs), or from the IBM Cloud Jobs page, which you access from your [Instances page](https://cloud.ibm.com/quantum/instances). From the session details you can determine the [maximum](#max-ttl) and [interactive](#ttl) time to live (TTL) values, its status, whether it's currently accepting jobs, and more. 
+
+Example:
+
+``` python
+from qiskit_ibm_runtime import QiskitRuntimeService
+
+service = QiskitRuntimeService()
+
+with Session(service=service, backend="ibmq_qasm_simulator") as session:
+  session.details()
+  ...
+```
+
+
 <span id="ends"></span>
 ## What happens when a session ends 
 
-A session ends by reaching its maximum timeout value or when it is
-manually closed by the user. Do not close a session until all jobs
-**complete**. See [Close a session](run-sessions#close) for details. After a session is closed, the following occurs:
-
--   Any jobs that are already running continue to run.
--   Any queued jobs remaining in the session are put into a failed state.
--   No further jobs can be submitted to the session.
--   The session cannot be reopened.
+A session ends by reaching its maximum timeout value or by exiting the context manager. See [Close a session](run-sessions#close) for details. What happens when a session closes depends on how the session ended.
+
+If the maximum timeout value was reached:
+    -   Any jobs that are already running are canceled.
+    -   Any queued jobs remaining in the session are put into a failed state.
+    -   No further jobs can be submitted to the session.
+    -   The session cannot be reopened.
+
+If the maximum timeout value has not been reached:    
+
+- When using Qiskit 12.3 or later releases:
+    -   The session is put in `Not accepting new jobs` status.
+    -   Any jobs that are already running continue to run.
+    -   Any queued jobs remaining in the session continue to run.
+    -   No further jobs can be submitted to the session.
+    -   The session cannot be reopened.
+    -   After all currently queued jobs are completed, the session is immediately closed and will not run any more jobs.
+
+- When using Qiskit 12.2 or previous releases:
+    -   Any jobs that are already running continue to run.
+    -   Any queued jobs remaining in the session are put into a failed state.
+    -   No further jobs can be submitted to the session.
+    -   The session cannot be reopened.
```

Comment:
OK - I'm updating this text but leaving the comment open while awaiting feedback about " Any running jobs continue to run   <-- **need to double check this**"  @mriedem , do you know? 

@jyu00 - does "closed vs canceled" only apply to the new changes, or does it also apply to the old content?

Reply:
> Any running jobs continue to run <-- need to double check this" @mriedem , do you know?

Yeah this is true, running jobs would continue to run. 

> does "closed vs canceled" only apply to the new changes, or does it also apply to the old content?

Only the new changes, in the sense that there was only the "cancel" behavior and not "close" prior to 0.13.

Reply:
Let me state that mapping the accepting_jobs field to "closed" is confusing when the closed state on the session in the server and API is something else. I get the meaning on qiskit though (it's closed to accepting more jobs). Since qiskit provides the abstraction layer that's fine. I'm just stating for the record that it gets confusing when talking to devs that knows how the server side works and closed there means the session is kaput. 😄

Reply:
@jyu00 , how do you cancel a session?  It looks to me like when you exit the context manager, the session is closed and if you use session.close(), the session is closed. If you cancel the first job, is the session canceled?

Reply:
There is a new `session.cancel()` method. This behaves like the old (pre 0.13.0) `close()` and puts all queued jobs in a failed state.